pkg/registry/tls.go

```diff
@@ -0,0 +1,92 @@
+package registry
```

Comment:
missing copyright

Also I'm tempted to call this registrytest to match the httptest pattern, what do you think? Or is the `registry` package already used exclusively for testing?

Reply:
registry is pretty much used only for testing. I think if we move this to another package it may be a lot less discoverable. It's not like this is dangerous to use in prod if you want a in memory registry for some reason.

Reply:
makes sense